examples/threejs_worker_brave_robot_gltf_ES6.js

```diff
@@ -191,7 +181,7 @@ export default function start(markerUrl, video, input_width, input_height, rende
     lasttime = now;
 
     if (!world) {
-      root.matrix = null;
```

Comment:
This was very bad: the result was an error complaining emty matrices:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'elements')
    at lr.multiplyMatrices (three.module.min.js:6:55816)
    at Dr.updateMatrixWorld (three.module.min.js:6:69980)
    at oc.updateMatrixWorld (three.module.min.js:6:70131)
    at rc.render (three.module.min.js:6:457337)
    at draw (threejs_worker_brave_robot_gltf_ES6.js:190:14)
    at tick (threejs_worker_brave_robot_gltf_ES6.js:203:5)
    at start (threejs_worker_brave_robot_gltf_ES6.js:208:3)
    at ARToolkitNFT_ES6_gltf_example.html:92:5
```
, but wasn't so obviuos to catch and solve. Leaving this comment for the future.